scripts/tests/run_python_test.py

```diff
@@ -32,7 +32,7 @@
 import click
 import coloredlogs
 from colorama import Fore, Style
-from py.metadata import Metadata, MetadataReader
```

Comment:
we have to get rid of the py prefix.

since we are building the metadata as a module properly now, please update build_python to also install this module and then something like `metadata_parser.metadata` should be a fully valid module.

Since we assume this will work in a general python environment, please also name it something like `matter_testing_support.metadata_parser` or `chip_testing_support.metadata_parser` or similar. That would make it distinct and unique enough to work well with others.


I would like to read code like:

```
from matter_testing_infrastructure.metadata_parser import Metadata, MetadataReader
```

Reply:
naming wise: I am trying to avoid support/helper/core/base/util and similar filler words, however realistically this is actually a grab-all package ... so maybe some form will be appropriate.

For now I like in order:
   - `matter_integration_tests`
   - `matter_testing_infrastructure`
   - `matter_testing`
   - `matter_test_support`
   - `matter_tests`
   
Happy with any reasonable name though.